docs/API/helpers/components.md

```diff
@@ -0,0 +1,116 @@
+# Distribute helpers as components
+
+Helpers can be distributed as components through a simple process to repackage them using our components (widgets) framework.
+
+## 1. Install the basic software
+
+Install the [Fliplet CLI](/Quickstart.html) on your computer before continuing with the rest.
+
+---
+
+## 2. Generate the component boilerplate
+
+Once you have installed the [Fliplet CLI](/Quickstart.html), open the shell of your computer and run the `fliplet create-widget <package> <name> --helper` command described below to create the initial structure of the component in a new directory with the same name as the `<name>` in the input.
+
+The command requires two inputs:
+
+- **Package name**: the unique "reverse domain name" to be used for your component, e.g. `com.example.decision-tree`
+- **Component name**: the user-friendly name of your component, e.g. `Decision tree`
+
+```bash
+fliplet create-widget "com.fliplet.decision-tree" "Decision tree" --helper
+```
+
+Upon running the command above, you should receive an output similar to this:
+
+```
+Creating new widget Decision tree to C:\Users\John\decision-tree using helper boilerplate
+Component has been successfully created.
+```
+
+---
+
+## 3. Configure the component
+
+Before we starting copying over the helper code to the component, it's best to open up the `widget.json` file you find in the created folder and configure the component by reviewing and tweaking the following properties:
```

Comment:
```suggestion
Before you start copying the helper code into the component, open the `widget.json` file you find in the created folder and configure the component by reviewing and tweaking the following properties:
```

Reply:
Fixed in e76af57d2d0528e92b4fcc468a78edbcb80a2d65